### ISSUE TITLE
Make animation of ion-refresher slower

### ISSUE THREAD

**Issue by duyetnk** [ionic-team/ionic#15425]
# Feature Request

**Ionic Info**
```
Ionic:

   ionic (Ionic CLI)  : 4.1.1 (C:\Users\naugh\AppData\Roaming\npm\node_modules\ionic)
   Ionic Framework    : ionic-angular 3.9.2
   @ionic/app-scripts : 3.1.11

Cordova:

   cordova (Cordova CLI) : 8.0.0
   Cordova Platforms     : android 7.0.0, browser 5.0.1
   Cordova Plugins       : cordova-plugin-ionic-webview 1.2.1, (and 16 other plugins)

System:

   NodeJS : v8.10.0 (C:\Program Files\nodejs\node.exe)
   npm    : 5.6.0
   OS     : Windows 10
```

**Describe the Feature Request**
I want the animation of ion-refresher slower when user pull down,

**Describe Preferred Solution**
i want it slower than speed of user's finger,

**Describe Alternatives**
Can you guy add new input property such as "speed" in ion-refresher so i can specify the speed i want.  (if possible). Example:

```
<ion-refresher speed="80" pullMax="100" pullMin="60" (ionRefresh)="loadHomePage($event)">
</ion-refresher>
```

**Related Code**
Here is the example I found in ionic v1, it is perfect what I want
https://codepen.io/anon/pen/ZYYVgX?editors=101

**Comment by mhartington** [comment]
What do you mean by speed? There's 

```
  /**
   * Time it takes the refresher to to snap back to the `refreshing` state. Defaults to `280ms`.
   */
  @Prop() snapbackDuration = '280ms';
```

Can you test this to see if this is what you need?

**Comment by duyetnk** [comment]
No, i mean speed of animation when content page pull down with user's finger (or cursor).
Not snapbackDuration (it's pull up)

i want it slower, by now it equal to the speed of user's finger (or cursor). => It does not look smooth in android.

**Comment by brandyscarney** [comment]
Thanks for the issue! This has been added by the following PR: https://github.com/ionic-team/ionic/pull/16697

It will be in the `4.4.0` release and used like the following:

```
<ion-refresher slot="fixed" pull-factor="0.5" pull-min="100" pull-max="200">
  <ion-refresher-content></ion-refresher-content>
</ion-refresher>
```
